### ISSUE TITLE
volume default in jr6_txt conversion

### ISSUE THREAD

**Issue by moonshoes87** [PmagPy/PmagPy#398]
The current default is 2.5 * 1e-6.  In most of the other conversions, the default volume is 2.5**3 * 1e-6.   @ltauxe , Is that correct?

**Comment by moonshoes87** [comment]
Same question with jr6_jr6 magic

**Comment by njarboe** [comment]
2.5 * 1e-6 is definitely wrong. Thanks for catching that.

2.5**3 * 1e-6 for a default would be a one inch cube.
For my work I would use a one inch diameter core, one inch in length as the default: 
3.14*(1.25*1.25)*2.5*1e-6

But if all the others use cube defaults and Lisa thinks that is best, I'll agree with her.
Having them all the same is key.

**Comment by ltauxe** [comment]
right.  2.51e-6 is totally wrong.  I would go with the standard core as a default - the cubes can have many sizes. 
lisa

**Comment by moonshoes87** [comment]
So, is this correct?  Input is in cc, and the default should be:

3.14 * 1.25 * 1.25 * 2.51  (which is 12.3146875)

or use whatever number is inputted by the user.

Then multiply by 1e-6 to convert to m^3.

Is that correct?  If so, I will implement that in all of the conversion scripts that take a volume default.

**Comment by ltauxe** [comment]
yes - or round off to 12cc which is what i usually use.  
lisa

**Comment by moonshoes87** [comment]
Ok, I changed a bunch.  mini_magic currently is 10 cc -- should that last one be changed as well?

**Comment by ltauxe** [comment]
yes
